e2e_test/compaction/ingest_rows.slt

```diff
@@ -1,6 +1,6 @@
 statement ok
 CREATE SOURCE person
-    (id INTEGER, name VARCHAR, "email_address" VARCHAR, "credit_card" VARCHAR, city VARCHAR, state VARCHAR, "date_time" TIMESTAMP)
+    (id BIGINT, name VARCHAR, "email_address" VARCHAR, "credit_card" VARCHAR, city VARCHAR, state VARCHAR, "date_time" TIMESTAMP, "extra" VARCHAR)
```

Comment:
why change to bigint here

Reply:
The data type of `id` generated by nexmark Generator is 8-byte int, but some SQL use INTEGER and others use BIGINT. Generator used to serialize the data as json, and then the json would be parsed to the type specified by SQL, but now the generator generates `StreamChunk` directly, so we should use BIGINT here.